src/main/resources/static/api.yaml

```diff
@@ -1208,6 +1208,76 @@ paths:
               schema:
                 $ref: '#/components/schemas/ErrorResponse'
   # Sessions
+  /otpsession/verify:
+    post:
+      description: |
+        Request Otp for consent manager id.
+      requestBody:
+        required: true
+        content:
+          application/json:
+            schema:
+              $ref: '#/components/schemas/OtpVerificationRequest'
+          application/xml:
+            schema:
+              $ref: '#/components/schemas/OtpVerificationRequest'
+      responses:
+        200:
+          description: OK
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/OtpVerificationResponse'
+            application/xml:
+              schema:
+                $ref: '#/components/schemas/OtpVerificationResponse'
+        400:
+          description: Bad request(potentially bad request token)
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/ErrorResponse'
+            application/xml:
+              schema:
+                $ref: '#/components/schemas/ErrorResponse'
+        500:
+          description: Error(On server side)
+  /otpsession/permit:
+    post:
+      description: |
+        validate otp.
+      requestBody:
+        required: true
+        content:
+          application/json:
+            schema:
+              $ref: '#/components/schemas/OtpPermitRequest'
+          application/xml:
+            schema:
+              $ref: '#/components/schemas/OtpPermitRequest'
+      responses:
+        200:
+          description: OK
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/AuthResponse'
+            application/xml:
+              schema:
+                $ref: '#/components/schemas/AuthResponse'
+        401:
+          description: Unauthorized
+        400:
+          description: Bad request(potentially bad request token)
```

Comment:
What exactly you mean by that?

Reply:
I think it's incorrectly copied, will remove the 400 error section.